### PR TITLE
Adding typehinting to setuptools

### PR DESCRIPTION
**Pull request by matthewharvey-globality**
Adding `typehinting` to the `extras_require` as well as adding in `types-setuptools`.

### CODE REVIEW

**Review by marcglobality — APPROVED** [review]

**Comment by marcglobality** [comment]
@tomaszpietruszka-globality can you approve this?

File: setup.py [hunk 1]
```diff
@@ -52,6 +52,10 @@
             "flake8-logging-format",
             "globality-black",
         ],
+        "typehinting": [
+            "mypy",
+            "types-setuptools"
```
Comment (marcglobality) [inline setup.py:57]: can you add trailing comma here? or run gblack on this

Comment (matthewharvey-globality) [reply setup.py:57]: 🤣 - the irony is real.

**Review by tomaszpietruszka-globality — APPROVED** [review]

**Comment by matthewharvey-globality** [comment]
Added config to `build.json` then ran `pip install globality-build && globality-build build-gen local`

**Pull request merged**